test_reporting/kusto/setup.kql

```diff
@@ -43,12 +43,14 @@
 	    Error = tobool(cases["error"]),
         Summary = tostring(cases["summary"]),
         StartTime = todatetime(cases["start"]),
-        EndTime = todatetime(cases["end"])
+        EndTime = todatetime(cases["end"]),
+        DutCheckResult = tobool(cases["check"])
 }
 
 .create table TestCases (Feature: string, TestCase: string, ModulePath: string,
                          FilePath: string, StartLine: int, Runtime: double, Result: string,
-                         ReportId: string, Error: bool, Summary: string, StartTime: datetime, EndTime: datetime)
+                         ReportId: string, Error: bool, Summary: string, StartTime: datetime, EndTime: datetime,
+                         DutCheckResult: bool)
```

Comment:
It would be better to add a column for arbitrary customized message that can be inserted/updated by test cases. This will be much more flexible than adding a bool column with name specific to `DutCheckResult`.